packages/react-search-ui-views/src/MultiCheckboxFacet.js

```diff
@@ -43,9 +43,12 @@ function MultiCheckboxFacet({
       <div className="sui-multi-checkbox-facet__options-list">
         {options.length < 1 && <div>No matching options</div>}
         {options.map(option => {
-          const checked = !!values.find(value =>
-            deepEqual(option.value, value)
-          );
+          const checked = !!values.find(value => {
+            if (value && value.name && option.value.name === value.name)
+              return true;
```

Comment:
This is just for my edification as I get to know the codebase - but do we have Prettier or some linter config that's enforcing column width/this being on a separate line (as opposed to the line below where it's on the same line?)

EDIT: Ah actually I see the `format`/`lint-staged` scripts in the root package.json! Sweet!